proxy/proxy.go

```diff
@@ -350,11 +352,26 @@ func initLogging() {
 	}
 }
 
+var enablePprof bool
+
+func setupPprof() {
+	if !enablePprof {
+		return
+	}
+
+	go func() {
+		http.ListenAndServe("localhost:6060", nil)
+	}()
+}
+
 func main() {
 	initLogging()
 
+	flag.BoolVar(&enablePprof, "pprof", false, "enable pprof (visit http://localhost:6060/debug/pprof)")
```

Comment:
and ip to listen as well, would help to not only listen in localhost if it is a remote server where I want to debug